### PR TITLE
LibWeb: Apply the word-spacing css property to Node

### PR DESCRIPTION
**Pull request by kostyafarber**
This will let us start to begin applying this during text shaping.

### CODE REVIEW

File: Userland/Libraries/LibWeb/CSS/ComputedValues.h [hunk 1]
```diff
@@ -387,6 +388,7 @@ class ComputedValues {
     Vector<ShadowData> const& text_shadow() const { return m_inherited.text_shadow; }
     CSS::Positioning position() const { return m_noninherited.position; }
     CSS::WhiteSpace white_space() const { return m_inherited.white_space; }
+    Optional<CSS::LengthOrCalculated> word_spacing() const { return m_inherited.word_spacing; }
```
Comment (AtkinsSJ) [inline Userland/Libraries/LibWeb/CSS/ComputedValues.h:391]: I gave you bad advice before! `normal` is equivalent to `0`, so this doesn't need `Optional`.

File: Userland/Libraries/LibWeb/CSS/ComputedValues.h [hunk 2]
```diff
@@ -547,6 +549,7 @@ class ComputedValues {
         CSS::TextTransform text_transform { InitialValues::text_transform() };
         CSS::LengthPercentage text_indent { InitialValues::text_indent() };
         CSS::WhiteSpace white_space { InitialValues::white_space() };
+        Optional<CSS::LengthOrCalculated> word_spacing { InitialValues::word_spacing() };
```
Comment (AtkinsSJ) [inline Userland/Libraries/LibWeb/CSS/ComputedValues.h:552]: Same here

**Review by AtkinsSJ — CHANGES_REQUESTED** [review]
Looks good, just one thing

File: Userland/Libraries/LibWeb/CSS/ComputedValues.h (hunk 1 above)
Comment (kostyafarber) [reply Userland/Libraries/LibWeb/CSS/ComputedValues.h]: I'll remember for next time!

**Comment by kostyafarber** [comment]
@AtkinsSJ changes made :)

**Comment by kostyafarber** [comment]
Friendly ping @AtkinsSJ

**Pull request merged**